packages/components/src/Loader/Loader.css

```diff
@@ -0,0 +1,21 @@
+.loader {
+  position: relative;
+  padding: var(--space-base);
+}
+
+.determinate > div:last-child {
+  padding: 0 var(--space-larger);
+  background-color: var(--color-overlay) !important;
+}
+
+.overlay > div:last-child {
+  display: flex;
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: var(--color-overlay--dimmed);
```

Comment:
overlay--dimmed is not working because I used the wrong hyphenation syntax in the token names. Fix [here](https://github.com/GetJobber/atlantis/pull/599)

Reply:
merging in the fix doesn't resolve it, because I think components uses a generated css file, going to wait until that fix is in :)